bin/_tty

```diff
@@ -4,19 +4,6 @@
 #
 #/* vim: set filetype=sh : */
 
-show_help() {
-  echo "$(print_common_usage "$COMMAND")
-
-$(print_command_instructions)
-
-Options:
-  -e, --env   Environment name (default: development)
-  -h          Print usage
-
-Services:
-$(print_services)"
-}
-
```

Comment:
This change is not strictly necessary for adding long description support but this function is help related and completely wrong. Can move to a separate commit if deemed necessary.